packages/cosmic-swingset/src/launch-chain.js

```diff
@@ -139,7 +139,12 @@ export async function buildSwingset(
         );
         chainStorageEntries.push(entry);
         const [path] = entry;
-        const childEntries = callChainStorage('entries', path);
+        const childEntrySuffixes = callChainStorage('entries', path);
+        const childEntries = childEntrySuffixes.map(([nextSegment, value]) => {
+          const fullPath =
+            !path || path === '.' ? `${nextSegment}` : `${path}.${nextSegment}`;
```

Comment:
I believe `path === '.'` is invalid and I'm tempted to say we should disallow `path` from ever being empty in the initial `"get"`

```suggestion
          const fullPath =`${path}.${nextSegment}`;
```

Reply:
The interface doesn't currently allow enumerating top-level keys AFAICT, and that seems like a problem that could be addressed at some point—but in ignorance of this consumer. This anticipates and accommodates either a DNS-like `.` or an even more simple empty-string fix.

However, I think you're right about needing to require `STORAGE_PATH.CUSTOM` so it will be moot.